pwndbg/commands/telescope.py

```diff
@@ -39,7 +39,7 @@ def telescope(address=None, count=telescope_lines, to_string=False):
     ($sp by default)
     """
     address = int(address if address else pwndbg.regs.sp) & pwndbg.arch.ptrmask
-    count   = int(count) & pwndbg.arch.ptrmask
+    count   = max(int(count), 0) & pwndbg.arch.ptrmask
```

Comment:
it never worked like that, try it in the current version. B behaves like a count right now and that is preserved